### DISCUSSION TITLE
LSP keymapping

### DISCUSSION THREAD

**Discussion by gmdevz** [NvChad/NvChad#1325] [Q&A] [answered]
Hi I'm new to this neovim stuff and suddenly fell in love with it coz of this NvChad 😄 anyways about key mapping I would like to know how to use this "[d" and "d]" keymaps.. Thank you in advance 🤗 

![image](https://user-images.githubusercontent.com/62541303/178156192-7d9bf638-b4aa-4110-bcc2-f6cf2bb66db0.png)

**Comment by siduck** [comment, marked as answer]
check how to override our mappings, read docsssssssssssssssssssssssssssss!
